### ISSUE TITLE
Left stick click not recognised by sc-controller

### ISSUE THREAD

**Issue by afettouhi** [kozec/sc-controller#455]
For some reason the left stick click is no longer being recognized by sc-controller. The left stick click works fine in Steam with games Dead Island Definitive Edition etc. for sprinting with the character. I was playing The Division (uplay version) via Lutris with my Steam controller using sc-controller to emulate an Xbox 360 controller. Everything for work fine for the first 5 to 10 hours of gameplay. But then suddenly my character could no longer sprint, e.g. the left stick click action was no longer recognised. I enabled the test input mode in sc-controller and could see that left stick clicking no longer showed visual feedback. I have no idea what is causing this. The same issue happens also in Shadow Warrior 2 (GOG version) with Lutris.

**Comment by kozec** [comment]
That sounds mostly like HW issue, but just to check, can you try disabling emulation from the menu and  running `scc-daemon debug` in console? It may output some relevant error message when what you are describing happens.

**Comment by afettouhi** [comment]
I got it working again. I tried reinstall sc-controller and set my Steam controller up again sadly no change. But the controller works correctly with Steam games as mentioned before. Then I tried js-stick to see that would recognise the click function of the left stick and it did. Finally I tried to unplug the usb hub and plug it in again and now sc-controller sees the left stick click and it shows up in the test input mode. Playing Shadow warrior 2 again with my Steam controller. If this happens again I will do as you have suggested above report back. I am closing this issue until then.
